ITensorGPU/src/ITensorGPU.jl

```diff
@@ -1,13 +1,19 @@
 module ITensorGPU
 
+using NDTensors
+
 using Adapt
 using CUDA
 using CUDA.CUBLAS
 using CUDA.CUSOLVER
+if CUDA.runtime_version() ≥ v"12.0"
+  println(
+    "Warning: currently cuTENSOR fails with CUDA versions 12.0 and higher.\n To deal with this problem suggest reducing CUDA version to 11.8 using the command `CUDA.set_runtime_version!(\"11.8\")`",
+  )
```

Comment:
Maybe instead:
```julia
  println(
    "Warning: currently cuTENSOR does not support CUDA runtime versions 12.0 and higher.\n We suggest reducing the CUDA runtime version to below version 12 using the command `CUDA.set_runtime_version!`, for example `CUDA.set_runtime_version!(\"11.8\")`.",
  )
```
to be more agnostic about which version we recommend, since I don't think there is anything special about CUDA runtime version 11.8.